### DISCUSSION TITLE
XDG - Configs

### DISCUSSION THREAD

**Discussion by muescha** [pi0/config-dir#15] [General]
I came across this article [Clean your home folder ! discover XDG
](https://dev.to/ccoveille/clean-your-home-folder-discover-xdg-3ooh) from @ccoVeille while researching XDG config files. It delves into the concept and its application in addressing similar issues. Additionally, it introduces some tools:

- Here's a suggestion on configuring tools to utilize XDG folders:
  [xdg-ninja](https://github.com/b3nj5m1n/xdg-ninja)
  > This shell script checks your $HOME directory for unwanted files and directories.

- Another tool worth exploring is [antidot](https://github.com/doron-cohen/antidot).
  > It helps clean up your $HOME directory from pesky dotfiles.

**Comment by ccoVeille** [comment]
Hi there

Thanks for quoting my article on dev.to

Here is also an interesting page about xdg available on Arch wiki

https://wiki.archlinux.org/title/XDG_Base_Directory
